lib/src/util/react_util.dart

```diff
@@ -65,8 +56,7 @@ abstract class UiPropsMapView extends MapView
   // ----- builder_helpers.UiProps ----- //
 
   @override
-  bool get $isClassGenerated =>
-      throw UnimplementedError('@PropsMixin instances do not implement \$isClassGenerated');
+  bool get $isClassGenerated => true;
```

Comment:
This is needed when extending UiProps, otherwise the UiProps super-constructor will throw.

We do a similar override in non-generated subclasses of UiProps, such as `DomProps` and `GenericUiProps`.